lib/serverengine/daemon.rb

```diff
@@ -47,14 +47,23 @@ def initialize(server_module, worker_module, load_config_proc={}, &block)
       @chuser = @config[:chuser]
       @chgroup = @config[:chgroup]
       @chumask = @config[:chumask]
+
+      @pid = nil
+      @command_pipe = @config.fetch(:command_pipe, nil)
```

Comment:
Oh, yes.
My first code used it to pass `Server`, but current code doesn't (if needed, it's passed via config, and `Daemon` doesn't touch it).